CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this library adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- `Synchronizer.getFastestServers` function signature changed and does not require `Context` parameter anymore
```

Comment:
And now I realise that the entire change of this PR, to move some queries over Tor, is also not documented in the changelog.

As part of this, the fact that custom lightwalletd servers over VPNs like Tailscale stop working needs to be documented (see https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk/pull/1541#discussion_r2115539405).